salt/salt.spec

```diff
@@ -791,10 +806,14 @@ install -Dd -m 0755 %{buildroot}%{_sysconfdir}/logrotate.d/
 install -Dpm 0644 salt/cli/support/profiles/* %{buildroot}%{python3_sitelib}/salt/cli/support/profiles
 
 # Install Salt tests
-install -Dd -m 0750 %{buildroot}%{_datadir}/salt
-install -Dd -m 0750 %{buildroot}%{_datadir}/salt/tests
-cp -a tests/* %{buildroot}%{_datadir}/salt/tests/
-sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!/usr/bin/python3=' %{buildroot}%{_datadir}/salt/tests/runtests.py
+install -Dd %{buildroot}%{python3_sitelib}/salt-testsuite
+cp -r -a tests %{buildroot}%{python3_sitelib}/salt-testsuite/
```

Comment:
What do we need the nesting for? Can't we put the tests directly into `salt-testsuite/`?

Reply:
You're right, I actually did the same in the PoC and forgot about it